### DISCUSSION TITLE
Project's future

### DISCUSSION THREAD

**Discussion by arlgnv** [family/connectkit#386] [General]
Hi. Noticed slowing in project's development. Is it planned to develop it furher?

**Comment by lochie** [comment]
hey @alagunoff, overall connectkit is in a stable position, but we've got a few things in active development!

- updating [connectkit-next-siwe](https://github.com/family/connectkit/tree/main/packages/connectkit-next-siwe) to support [viem's siwe](https://viem.sh/docs/siwe/actions/verifySiweMessage) update (released yesterday) to we can remove all `ethers` dependencies.
- updating compatible `wagmi` and `@coinbase/wallet-sdk` versions to support [Coinbase Smart Wallet](https://www.smartwallet.dev/why).
- a few small patches to improve some UX and DX

we're also keeping our eyes on [React 19](https://react.dev/blog/2024/04/25/react-19) as we want to make sure we support the latest technology in the react ecosystem.

**Reply by arlgnv** [reply]
Got it. Glad to hear that project is not abandoned since I consider it the most pleasant-to-use 😃 Keep up a good work guys!
